src/utils.h

```diff
@@ -13,16 +13,32 @@ void rv_gettimeofday(struct timeval *tv);
 /* Retrieve the value used by a clock which is specified by clock_id. */
 void rv_clock_gettime(struct timespec *tp);
 
+#if RV32_HAS(JIT) && RV32_HAS(SYSTEM)
+
+typedef uint64_t rv_hash_dtype_t;
```

Comment:
How about `rv_hash_t`? The suffix `_t` implies the type.

Reply:
> How about `rv_hash_t`? The suffix `_t` implies the type.

I concern about it might be confused with the hash function or other data structure.

Reply:
Sure. I prefer to use `rv_hash_key_t` to prevent name conflict.